core/rawdb/schema.go

```diff
@@ -96,8 +96,9 @@ var (
 	// snapSyncStatusFlagKey flags that status of snap sync.
 	snapSyncStatusFlagKey = []byte("SnapSyncStatus")
 
-	// InclusionPreconfirmation virtual blocks
-	preconfBlockCursorKey = []byte("PreconfirmationBlockCursor")
+	// Inclusion preconfirmations
+	txPoolSnapshotPrefixKey   = []byte("txps")
+	preconfTxReceiptPrefixKey = []byte("ptxr")
```

Comment:
there is already a prefix with `t`, switch to `TxPoolSnap`. Let's switch `ptxr` to `PreconfTxR` for consistency.

Reply:
- `txPoolSnapshotPrefixKey = []byte("txps")` "TxPoolSnap" or even "TxPoolSnapshot" is fine, since single record is stored under this prefix

- `preconfTxReceiptPrefixKey = []byte("ptxr")` - here it is better to use shorter one, since there are multiple records stored under it